src/objects/nodes.ts

```diff
@@ -139,6 +172,7 @@ async function updateNodeBox(node: RunGraphNode, box: Graphics): Promise<Graphic
   const boxWidth = boxRight - boxLeft
   const boxHeight = config.styles.nodeHeight - config.styles.nodeMargin * 2
 
+  box.lineStyle(1, 0x0, 1, 2)
```

Comment:
Removing this in https://github.com/PrefectHQ/graphs/commit/aa7f73b50e41c69ccad446a4eb6ae3ab9508989d somehow slows everything down. Box rendering will likely change and use sprites but adding it back for now just to keep moving. No idea why removing this extra line around the boxes makes it excruciatingly slow

Reply:
??